pkg/controller/pxcrestore/controller.go

```diff
@@ -93,10 +89,12 @@ type ReconcilePerconaXtraDBClusterRestore struct {
 func (r *ReconcilePerconaXtraDBClusterRestore) Reconcile(ctx context.Context, request reconcile.Request) (reconcile.Result, error) {
 	log := logf.FromContext(ctx)
 
-	rr := reconcile.Result{}
+	rr := reconcile.Result{
+		RequeueAfter: time.Second * 5,
+	}
```

Comment:
honestly I'm not happy to depend on `RequeueAfter` but I guess there's no other way

Reply:
There is a way to not depend on `RequeueAfter`, but it will take more time to implement. I would like to do it in a separate PR.